src/components/ButtonPanel.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import Button from './Button'
+const grey = 'rgb(224, 224, 224)'
+function ButtonPanel (props) {
```

Comment:
let's put a space between the `import` statements and the succeeding lines of code